### ISSUE TITLE
Failed to load Intermix component SVGs

### ISSUE THREAD

**Issue by cschol** [stoermelder/vcvrack-packone#78]
When loading the plugin version 1.4.0, the following warnings are issued: 

```
[0.814 warn src/window.cpp:75] Failed to load SVG /home/cschol/src/Rack-1.0-Home/plugins-v1/Stoermelder-P1/res/Components/IntermixButton.svg
[0.814 warn src/window.cpp:75] Failed to load SVG /home/cschol/src/Rack-1.0-Home/plugins-v1/Stoermelder-P1/res/Components/IntermixButton1.svg
```

I don't see the SVGs in the repository.

**Comment by stoermelder** [comment]
Did you check the commit I referenced?
https://github.com/stoermelder/vcvrack-packone/tree/f9c8c9f1fd9dc2cc80671f8c82fd27f1bfda8579
I renamed the files in a later commit but this should not break older commits as far as I know?

**Comment by cschol** [comment]
The SVGs are there. I missed them because I didn't look in the `components` directory.

I know what the issues is:

`Components` is upper-case in the C++ code:

```
src/Intermix.cpp
594:            addFrame(APP->window->loadSvg(asset::plugin(pluginInstance, "res/Components/IntermixButton.svg")));
595:            addFrame(APP->window->loadSvg(asset::plugin(pluginInstance, "res/Components/IntermixButton1.svg")));
```

But the directory is `components` on my Linux system.

**Comment by stoermelder** [comment]
Yeah, I thought the same some seconds ago...

**Comment by cschol** [comment]
Yeah, you have to be careful with case-sensitive systems. This problem won't show on Windows. Let me know when it is fixed and I will integrate.

**Comment by stoermelder** [comment]
Thank you for your assistance, Christoph!
Fixed in 613036f4ab2042e3b6e3a7ed7399ee0725860352.
